python/pylibraft/pylibraft/solver/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# =============================================================================
+# Copyright (c) 2022-2024, NVIDIA CORPORATION.
```

Comment:
This is a new file. Please make sure copyrights for new files only contain the current year